content/ko/docs/tasks/access-application-cluster/web-ui-dashboard.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 웹 UI (대시보드)
+content_template: templates/concept
+weight: 10
+card:
+  name: tasks
+  weight: 30
+  title: Use the Web UI Dashboard
```

Comment:
This title is not localized. Intentional?

Reply:
Intentional! Maybe this should be left in English.